src/model_theory/semantics.lean

```diff
@@ -603,7 +603,20 @@ variables (L)
 /-- The complete theory of a structure `M` is the set of all sentences `M` satisfies. -/
 def complete_theory : L.Theory := { φ | M ⊨ φ }
 
-variables {L}
+variable (N)
+
+/-- Two structures are elementarily equivalent when they satisfy the same sentences. -/
+def elementarily_equivalent : Prop := L.complete_theory M = L.complete_theory N
+
+localized "notation A ` ≅[`:25 L `] ` B := first_order.language.elementarily_equivalent L A B"
```

Comment:
```suggestion
localized "notation A ` ≅[`:25 L `] ` B:xy := first_order.language.elementarily_equivalent L A B"
```
Something like this should give this the right precedence, for suitable `xy` (`50` or so). Copy whatever is done for other `notation` commands.